DESCRIPTION

```diff
@@ -79,7 +79,8 @@ Suggests:
     RANN,
     multtest,
     MsBackendMgf,
-    signal
+    signal,
+    mgcv
```

Comment:
Hi, I see a new package dependency here. Where is it used ? 

I see an explicit `library(mgcv)` in `test_do_adjustRtime-functions.R`,
but I suspect it us used somewhere in `xcms/R/*.R` as well. 

I'd love to be able to handle `DESCRIPTION` and `NAMESPACE` through roxygen2 one day, and for that we'd need to start annotations 
like `#' @importFrom MsCoreUtils ndotproduct`  
Does that make sense for https://github.com/sneumann/xcms/blob/805ac3f8ba4499d41068eb2748c3ff5a4dbc752a/R/functions-Params.R#L278 ?

Or is the package really only needed in the `test_do_adjustRtime-functions.R` ?

Reply:
Also chiming in here - I absolutely want to avoid any additional (potentially unnecessary) dependency. We can have the `mgcv` either in Suggests or Enhances since we're **not** importing any functionality. We're loading the namespace specifically in the function, and only if that method is used. If the package is not installed an error is thrown that, to use the functionality, the `mgcv` package needs to be installed.

Reply:
Yes, maybe it's good (also good practice in general) to already add the `@importFrom` roxygen directives into the documentation. @philouail , can you please add them to your new functions?

Reply:
I think it should be good now, just had to move one package around to make it more clear + added the now published `force_sorted()`, function from `MsCoreUtils`.